src/components/Navbar.tsx

```diff
@@ -107,20 +108,52 @@ const loadPageURL = (menuItem: any) => {
  */
 export default function Navbar() {
   const [selectedKeys, setselectedKeys] = useState<Array<string>>([]);
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false);
 
   useEffect(() => {
     setselectedKeys([loadPageURL(window.location.pathname.substring(1))]);
   }, []); // Empty dependency to make it run once
 
+  const menuItems = [
```

Comment:
Okay to move it outside if you want to call the menu from multiple places, but doing so in the same component and then calling it twice loads the whole menu twice.

![image](https://github.com/user-attachments/assets/a45bc2b4-87a1-412f-aa9b-2d4a2877b72e)